{"uid":"31a78bb2fce1b06f","name":"test_user_can_go_to_checkout_complete_page[problem_user-secret_sauce]","fullName":"tests.test_TC_016_01_return_to_catalog_after_order.TestSample#test_user_can_go_to_checkout_complete_page","historyId":"a0ae2b1f654c22ef377249a2cfb07fcf","time":{"start":1671195051912,"stop":1671195053123,"duration":1211},"description":"Checks whether a user can navigate to the checkout complete page of a website\n        after logging in, adding items to cart, navigating to the cart page,\n        filling out the shipping information on the first checkout page, and navigating to the second checkout page.\n\n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        ","descriptionHtml":"<p>Checks whether a user can navigate to the checkout complete page of a website<br />\nafter logging in, adding items to cart, navigating to the cart page,<br />\nfilling out the shipping information on the first checkout page, and navigating to the second checkout page.</p>\n<pre><code>    Args:\n        d: WebDriver\n        username: used to specify the username of the user\n        password: used to specify the password of the user\n</code></pre>\n","status":"skipped","statusMessage":"XFAIL\n\nAssertionError: Expected link -step-two not found in URL https://www.saucedemo.com/checkout-step-one.html","statusTrace":"self = <SauceDemoProject.tests.test_TC_016_01_return_to_catalog_after_order.TestSample object at 0x7f87acdccfd0>\nd = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b0e48bb8afa0122557f2ebdac8a22f7b\")>\nusername = 'problem_user', password = 'secret_sauce'\n\n    @pytest.mark.parametrize(\n        \"username, password\",\n        [\n            (\"standard_user\", \"secret_sauce\"),\n            (\"performance_glitch_user\", \"secret_sauce\"),\n            pytest.param(\n                \"problem_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n            pytest.param(\n                \"locked_out_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n        ],\n    )\n    def test_user_can_go_to_checkout_complete_page(self, d, username, password):\n        \"\"\"Checks whether a user can navigate to the checkout complete page of a website\n        after logging in, adding items to cart, navigating to the cart page,\n        filling out the shipping information on the first checkout page, and navigating to the second checkout page.\n    \n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        \"\"\"\n>       self.test_user_can_go_to_checkout_2_page(d, username, password)\n\ntests/test_TC_016_01_return_to_catalog_after_order.py:177: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_TC_016_01_return_to_catalog_after_order.py:148: in test_user_can_go_to_checkout_2_page\n    page.should_be_checkout_2_page()\npages/checkout_2_page.py:15: in should_be_checkout_2_page\n    self.should_be_link(Chckout2PageSrc.LINK)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.checkout_2_page.CheckoutPage_2 object at 0x7f87abc562f0>\nlink = '-step-two'\n\n    def should_be_link(self, link):\n        \"\"\"Check that the link provided is in the current URL of the browser.\n    \n        Args:\n            link: link browser\n        \"\"\"\n        assert (\n>           link in self.browser.current_url\n        ), f\"Expected link {link} not found in URL {self.browser.current_url}\"\nE       AssertionError: Expected link -step-two not found in URL https://www.saucedemo.com/checkout-step-one.html\n\npages/base_page.py:350: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"d","time":{"start":1671195007843,"stop":1671195008743,"duration":900},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"browser","time":{"start":1671195007843,"stop":1671195007843,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"g","time":{"start":1671195008744,"stop":1671195008930,"duration":186},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"Checks whether a user can navigate to the checkout complete page of a website\n        after logging in, adding items to cart, navigating to the cart page,\n        filling out the shipping information on the first checkout page, and navigating to the second checkout page.\n\n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        ","status":"skipped","statusMessage":"XFAIL\n\nAssertionError: Expected link -step-two not found in URL https://www.saucedemo.com/checkout-step-one.html","statusTrace":"self = <SauceDemoProject.tests.test_TC_016_01_return_to_catalog_after_order.TestSample object at 0x7f87acdccfd0>\nd = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b0e48bb8afa0122557f2ebdac8a22f7b\")>\nusername = 'problem_user', password = 'secret_sauce'\n\n    @pytest.mark.parametrize(\n        \"username, password\",\n        [\n            (\"standard_user\", \"secret_sauce\"),\n            (\"performance_glitch_user\", \"secret_sauce\"),\n            pytest.param(\n                \"problem_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n            pytest.param(\n                \"locked_out_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n        ],\n    )\n    def test_user_can_go_to_checkout_complete_page(self, d, username, password):\n        \"\"\"Checks whether a user can navigate to the checkout complete page of a website\n        after logging in, adding items to cart, navigating to the cart page,\n        filling out the shipping information on the first checkout page, and navigating to the second checkout page.\n    \n        Args:\n            d: WebDriver\n            username: used to specify the username of the user\n            password: used to specify the password of the user\n        \"\"\"\n>       self.test_user_can_go_to_checkout_2_page(d, username, password)\n\ntests/test_TC_016_01_return_to_catalog_after_order.py:177: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_TC_016_01_return_to_catalog_after_order.py:148: in test_user_can_go_to_checkout_2_page\n    page.should_be_checkout_2_page()\npages/checkout_2_page.py:15: in should_be_checkout_2_page\n    self.should_be_link(Chckout2PageSrc.LINK)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.checkout_2_page.CheckoutPage_2 object at 0x7f87abc562f0>\nlink = '-step-two'\n\n    def should_be_link(self, link):\n        \"\"\"Check that the link provided is in the current URL of the browser.\n    \n        Args:\n            link: link browser\n        \"\"\"\n        assert (\n>           link in self.browser.current_url\n        ), f\"Expected link {link} not found in URL {self.browser.current_url}\"\nE       AssertionError: Expected link -step-two not found in URL https://www.saucedemo.com/checkout-step-one.html\n\npages/base_page.py:350: AssertionError","steps":[],"attachments":[{"uid":"b583e574022e56d6","name":"Screenshot","source":"b583e574022e56d6.png","type":"image/png","size":87399}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"g::0","time":{"start":1671195070569,"stop":1671195070629,"duration":60},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(raises=<class 'AssertionError'>)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_TC_016_01_return_to_catalog_after_order"},{"name":"subSuite","value":"TestSample"},{"name":"host","value":"fv-az574-63"},{"name":"thread","value":"2750-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_TC_016_01_return_to_catalog_after_order"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"password","value":"'secret_sauce'"},{"name":"username","value":"'problem_user'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.xfail(raises=<class 'AssertionError'>)"]},"source":"31a78bb2fce1b06f.json","parameterValues":["'secret_sauce'","'problem_user'"]}